index.html

```diff
@@ -32,7 +32,139 @@ <h2 class="first-name software-dev">I'm a software developer</h2>
+  <div class="gray-container"></div>  
+  <div class="multi-post">
+    <h3 class="multi-text-color">Multi-Post Stories Gain+Glory</h3>
+  <ul class="language display-flex">
+    <li class="language-background">Ruby on rails</li>
+    <li class="language-background">css</li>
+    <li class="language-background" >JavaScript</li>
+    <li class="language-background">html</li>    
+  </ul>
+  <button class="see-project">See Project</button>
+</div>
+</div>
+
+<div class="project-1 grid-project display-flex">
+  <div class="gray-container"></div>  
+  <div class="multi-post">
+    <h3 class="multi-text-color">Multi-Post Stories Gain+Glory</h3>
+  <ul class="language display-flex">
+    <li class="language-background">Ruby on rails</li>
+    <li class="language-background">css</li>
+    <li class="language-background" >JavaScript</li>
+    <li class="language-background">html</li>    
+  </ul>
+  <button class="see-project">See Project</button>
+</div>
+</div>
+
+<div class="project-1 grid-project display-flex">
+  <div class="gray-container"></div>  
+  <div class="multi-post">
+    <h3 class="multi-text-color">Multi-Post Stories Gain+Glory</h3>
+  <ul class="language display-flex">
+    <li class="language-background">Ruby on rails</li>
+    <li class="language-background">css</li>
+    <li class="language-background" >JavaScript</li>
+    <li class="language-background">html</li>    
+  </ul>
+  <button class="see-project">See Project</button>
+</div>
+</div>
+
+<div class="project-1 grid-project display-flex">
+  <div class="gray-container"></div>  
+  <div class="multi-post">
+    <h3 class="multi-text-color">Multi-Post Stories Gain+Glory</h3>
+  <ul class="language display-flex">
+    <li class="language-background">Ruby on rails</li>
+    <li class="language-background">css</li>
+    <li class="language-background" >JavaScript</li>
+    <li class="language-background">html</li>    
+  </ul>
+  <button class="see-project">See Project</button>
+</div>
+</div>
+
+<div class="project-1 grid-project display-flex">
+  <div class="gray-container"></div>  
+  <div class="multi-post">
+    <h3 class="multi-text-color">Multi-Post Stories Gain+Glory</h3>
+  <ul class="language display-flex">
+    <li class="language-background">Ruby on rails</li>
+    <li class="language-background">css</li>
+    <li class="language-background" >JavaScript</li>
+    <li class="language-background">html</li>    
+  </ul>
+  <button class="see-project">See Project</button>
+</div>
+</div>
+</section>
+
+<!-- About me Session -->
+<section class="about-container">
+  <h2 class="about-me">About Me</h2>
+     <p class="about-message"> I can help you build a product, feature or website 
+         Look through some of my work and experience! If you like what you see and have a project you need coded, 
+          don't hestiate to contact me.</p>
+  <button class="see-project add-bottom-margin">Get My Resume</button>  
+</section>
```

Comment:
- [ ] You are missing the background decorations as we have in the template. Kindly fix that.

- [ ] Kindly ensure your paragraph font spacing are similar to what we have on the template.

- [ ] Kindly ensure your button is similar to what we have on the template, as highlighted earlier

  Your Implementation | Figma Design
  :---:|:---:
  ![ezea-about-html](https://user-images.githubusercontent.com/36542986/164949261-b9d241e5-9f9d-4553-9971-ac621562e825.jpg) | ![ezea-about-figma](https://user-images.githubusercontent.com/36542986/164949258-b338c79f-5bda-4214-bb2f-a8c45a65f9bc.jpg)

Reply:
good, done